database/insert_data.sh

```diff
@@ -17,8 +17,6 @@ fi
 PID=$!
 wait $PID
 
-yarn ts-node ./database/updateRowPoliciesCLI.ts &
```

Comment:
No longer needed -- we run `updatePolicies` at the end of snapshot load, plus whenever the server restarts.